bluetooth_mesh/messages/util.py

```diff
@@ -452,10 +453,8 @@ def camelcase(field_name):
 
 
 def snakecase(camel_input):
-    words = re.findall(
-        r"[A-Z]?[a-z\d]+|[A-Z]{2,}(?=[A-Z][a-z]|\d|\W|$)|\d+", camel_input
```

Comment:
Please double check this with @przemyslawfierek-silvair

Reply:
@michallowasrzechonek-silvair I checked with him, also I added few cases for DebugV2, and everything works with `{1, }` instead of `{2, }`